### DISCUSSION TITLE
Suggestion: option to choose what time when selecting the "SortyBy" option

### DISCUSSION THREAD

**Discussion by arthurdeka** [JackhammerYT/RedDownloader#17] [Ideas]
first of all, congratulations to the developers, you're doing an amazing work!
my suggestion is just as the title says, give the option to choose the time when sorting by Hot such as "today", "this week", "this moth", etc.

**Comment by kents00** [comment]
Hello, buddy. Thank you for creating this library. I'd like to suggest that every video that I download be included in the title of the post.
Example:
[Include Title Name ].mp4 ( it's like a [pytube](https://pytube.io/en/latest/user/quickstart.html) or something )

**Comment by Jackhammer9** [comment]
Greetings 

Every few months i work on this library for a week to add new features ill make sure to include this in new update.
